scripts/env.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if (( $# != 1 )); then
+    echo "Usage: source env.sh aws_profile_name"
+    echo "       where aws_profile_name is an AWS CLI named profile"
+    echo "       https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html"
+    exit 1
+else
+    export AWS_PROFILE=$1
+
+    AWS_ACCESS_KEY_ID=$(aws configure get aws_access_key_id --profile "$AWS_PROFILE")
+    AWS_SECRET_ACCESS_KEY=$(aws configure get aws_secret_access_key --profile "$AWS_PROFILE")
+    AWS_REGION=$(aws configure get region --profile "$AWS_PROFILE" || echo "$AWS_DEFAULT_REGION")
```

Comment:
Is there some magic that uses the region setting in the config to set AWS_DEFAULT_REGION?

Reply:
Hmm, I don't know, good question.

Reply:
It looks like AWS_REGION is the envvar to use now. AWS_DEFAULT_REGION is checked if it is not set. Version 2 of the AWS CLI and the python boto3 library both use AWS_REGION, where prior versions were inconsistent.